models/accounts.go

```diff
@@ -6,13 +6,15 @@ import (
 )
 
 type Account struct {
-	ID             string    `json:"id" bson:"_id,omitempty"`
-	Email          *string   `json:"email" bson:"email,omitempty"`
-	Name           *string   `json:"name" bson:"name,omitempty"`
-	IsInMobileBeta bool      `json:"is_in_mobile_beta" bson:"is_in_mobile_beta,omitempty"`
-	Hash           *[]byte   `json:"hash" bson:"hash,omitempty"`
-	Token          string    `json:"token" bson:"token,omitempty"`
-	ValidUntil     time.Time `json:"valid_until" bson:"valid_until,omitempty"`
+	ID              string    `json:"id" bson:"_id,omitempty"`
+	Email           *string   `json:"email" bson:"email,omitempty"`
+	Name            *string   `json:"name" bson:"name,omitempty"`
+	Hash            *[]byte   `json:"hash" bson:"hash,omitempty"`
+	ValidationToken string    `json:"validation_token" bson:"validation_token,omitempty"`
+	IsValidate      bool      `json:"is_validate" bson:"is_validate"`
```

Comment:
```suggestion
	Validated     bool      `json:"is_validate" bson:"is_validate"`
```
at least for the model